plugin_store/api/__init__.py

```diff
@@ -65,15 +66,28 @@ async def index():
     return INDEX_PAGE
 
 
+# TODO: move these enums somewhere to deduplicate them
+class SortOption(StrEnum):
+    desc = auto()
+    asc = auto()
+    NONE = auto()
```

Comment:
Can we just use a default of `desc` in functions using that and get rid of `NONE`? it essentially means `desc` as this is the default hardcoded later.